programs/multisig/src/lib.rs

```diff
@@ -239,6 +252,20 @@ pub struct ExecuteTransaction<'info> {
     transaction: ProgramAccount<'info, Transaction>,
 }
 
+#[derive(Accounts)]
+pub struct DropTransaction<'info> {
```

Comment:
I guess the tradeoff here is that you would save some SOL vs. getting the option to query what transactions you executed like last month. I suppose you don't *have* to call the drop transaction from the UI if you want to keep the transaction around

Reply:
Even if the transaction account would be dropped it still should be possible to know what transactions were exected? (of course, archive node is required)

Reply:
Also might be worth emitting an event in the future to make indexing this easier.

Reply:
Ah makes sense.